authors/apps/authentication/tests/test_views.py

```diff
@@ -32,9 +32,18 @@ def setUp(self):
         self.valid_email = {"user": {"email": "dude1@gmail.com"}}
         self.empty_email = {"user": {"email": ""}}
 
+        self.send_password_reset_email = '/api/users/password_reset/'
```

Comment:
PEP8 guidelines recommend using a different way for formatting dictionaries, lists, e.t.c to enhance legibility. 
I'd recommend you follow the [PEP8 Guidelines](https://www.python.org/dev/peps/pep-0008/#indentation).
You can look at this [StackOverflow question](https://stackoverflow.com/questions/3985563/python-best-formatting-practice-for-lists-dictionary-etc) with provides guidance on how to do it.